service/installer/helm/helm.go

```diff
@@ -365,7 +371,9 @@ func (i *HelmInstaller) Install(event eventerspec.DeploymentEvent) error {
 	var installCommand []string
 	{
 		installCommand = append(installCommand, "upgrade", "--install")
-		installCommand = append(installCommand, forceArg)
+		if forceArg != "" {
+			installCommand = append(installCommand, forceArg)
```

Comment:
This is fixing another bug in draughtsman; it introudced blank space in command arguments and give an error like below. 
```
D 03/02 12:14:40 ran helm command | draughtsman/service/installer/helm/helm.go:203 | name=install | stderr=Error: "helm upgrade" requires 2 arguments
Usage:  helm upgrade [RELEASE] [CHART] [flags]
 | stdout=
U 03/02 12:14:40 helm error: error output: Error: "helm upgrade" requires 2 arguments
```